LongWarOfTheChosen/Src/LW_Overhaul/Classes/LWTacticalMissionUnitSpawner.uc

```diff
@@ -499,7 +504,11 @@ static function LoadLiaisonFromOutpost(XComGameState_LWOutpost Outpost,
 		{
 			ItemState = XComGameState_Item(NewGameState.CreateStateObject(class'XComGameState_Item', ItemReference.ObjectID));
 			NewGameState.AddStateObject(ItemState);
+			//Issue #159, this is needed now for loading units from avenger to properly update gamestate.
+			ItemState.BeginTacticalPlay(NewGameState);
```

Comment:
Is this one needed, since we're doing it in the previous submitted game state as well? Also, if it is, I'd appreciate if you could update `CreateStateObject()` as per earlier comment.

Reply:
This change does seem to be needed as well as it serves to apply item abilities. I'll make the correction to use `ModifyStateObject()` instead.

Reply:
done